src/Stratis.Bitcoin.Features.SmartContracts/ReflectionExecutor/Controllers/SmartContractsController.cs

```diff
@@ -78,7 +78,16 @@ public class SmartContractsController : Controller
             this.smartContractTransactionService = smartContractTransactionService;
         }
 
-        [Route("code")]
+        /// <summary>
+        /// Gets the bytecode for a smart contract as a hexadecimal string. The bytecode is decompiled to
+        /// C# source, which is returned as well. Be aware, it is the bytecode which is being executed,
+        /// so this is the "source of truth".
+        /// </summary>
+        ///
+        /// <param name="address">The address of the smart contract to retrieve as bytecode and C# source.</param>
+        ///
+        /// <returns>A response object containing the bytecode and the decompiled C# code.</returns>
+        [Route("SC-bytecode")]
```

Comment:
You don't really need to have SC in the endpoint as the url already looks like so: `/api/smartcontracts/code`

Reply:
If SC was omitted, it should probably be /api/smartcontracts/bytecode
Why is having "the endpoint all lowercase is better."? Is it a standard? The capitalisation was just to indicate abbreviation. In general, the endpoints were confusingly named, which is my reason for changing - are the "clients" our internal programmes or things written externally.

Reply:
Lowercase is how we have in in all our endpoints, so it's better to keep it that way for consistency.

The clients are our internal programs for now, like the wallets and the block explorers, but in time, we'll see more. I'm not saying they should never change, but when they do, people need to be notified.

Reply:
We can revert to lower case no probs. If we can use this as an attempt to improve the Swagger API, it will help a great deal. When going through it with Jordon and Rowan, design inconsistencies did come up, and they can be smoothed out along with this round of documentation. If it's all internal, it shouldn't be hard to fix up if changes are made, should it?

Reply:
+1 to Jeremy's suggestions. Remove 'SC-' prefix.

Reply:
Actually, I'd suggest not changing this at all. The endpoint returns both bytecode and C#, so naming it `bytecode` is incorrect.